### PR TITLE
🚀Preview Release

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Release Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

- [x] The **_base(target)_** branch for this pull request is a **_release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: release/v<major>.<minor>.<patch>
      Example: release/v1.2.3
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_release_** branch with the correct naming syntax in the following example.
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: release/v<major>.<minor>.<patch>
    Example: release/v1.2.3
    ```
  </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title is set to _**🚀Preview Release**_.

- [x] A `🚀Preview Release` label has been added to this pull request. 

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] A QA issue has been created and manual QA testing has been performed for the changes being released _(if applicable)_.

### CODE REVIEW

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md [hunk 1]
```diff
@@ -46,20 +46,7 @@ As with all software, there is always a chance for issues and bugs, especially f
...
-
----
-
-<h2 style="font-weight:bold" align="center">Internal Changes ⚙️</h2>
-<h5 align="center">(Changes that do not affect users.  Not breaking changes, new features, or bug fixes.)</h5>
-
----
-
-<h2 style="font-weight:bold" align="center">Nuget/Library Updates 📦</h2>
+   - A hotfix uses a hot-fix branch, not a preview feature branch.  This is why this was a bug.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md:49]: Grammar
```suggestion
   - A hotfix uses a hot-fix branch, not a preview feature branch, which is why this was a bug.
```

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.12.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<div align="center">
+
+As with all software, there is always a chance for issues and bugs, especially for preview releases, which is why your input is greatly appreciated. 🙏🏼
+</div>
+
+---
+
+<h2 style="font-weight:bold" align="center">Bug Fixes 🐛</h2>
+
+1. [#113](https://github.com/KinsonDigital/CICD/issues/113) - Fixed a bug where **Preview** and **Production** PR status checks were incorrectly reporting milestone state.
```
Comment (kselena) [inline Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.12.md:16]: Grammar
```suggestion
1. [#113](https://github.com/KinsonDigital/CICD/issues/113) - Fixed a bug where **Preview** and **Production** PR status checks were reporting the incorrect milestone.
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.12.md (hunk 2 above)
Comment (CalvinWilkinson) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.12.md:16]: This bug has nothing to do with reporting the incorrect milestone.
It had to do with reporting the incorrect _**state**_ of the milestone.

But now that I read it, I think it should be:
```suggestion
1. [#113](https://github.com/KinsonDigital/CICD/issues/113) - Fixed a bug where **Preview** and **Production** PR status checks were reporting the incorrect state of the milestone.
```

Thoughs?

Comment (kselena) [reply Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.12.md:16]: I like "reporting the incorrect state of the milestone."  Let's go with it.

**Pull request merged**